### PR TITLE
add site 404

### PR DESCRIPTION
**Pull request by Fercas123**

### CODE REVIEW

File: site/src/theme/notfound.module.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+}
+.notFoundContent:before {
+  content: "";
+  border: 1px solid;
+  border-color: var(--uitkAccordion-section-borderColor, var(--uitk-separable-secondary-borderColor));
```
Comment (libertymayc) [inline site/src/theme/notfound.module.css:29]: probably shouldn't be this first token?

Comment (Fercas123) [inline site/src/theme/notfound.module.css:29]:
```suggestion
  border-color: var(--uitk-separable-secondary-borderColor);
```

File: site/src/theme/NotFound.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
```
Comment (JoanaMMoreira) [inline site/src/theme/NotFound.js:1]: Very minor but could this be a `tsx` file instead?

File: site/src/theme/NotFound.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import Link from "@docusaurus/Link";
+import { PageMetadata } from "@docusaurus/theme-common";
+import { translate } from "@docusaurus/Translate";
+
+export default function NotFound() {
+  return (
+    <>
+      <PageMetadata
+        title={translate({
+          id: "theme.NotFound.title",
```
Comment (JoanaMMoreira) [inline site/src/theme/NotFound.js:13]: Is this `id` required?

**Review by libertymayc — APPROVED** [review]

File: site/src/theme/NotFound.js (hunk 3 above)
Comment (JoanaMMoreira) [reply site/src/theme/NotFound.js:13]: We may not need `translate` either

**Review by JoanaMMoreira — APPROVED** [review]
🚀

**Pull request merged**